### PR TITLE
Add v1.3.0 examples

### LINKED ISSUE #11: Rewrite readme.MD and add docs.
Add documentation and change the readme.MD to align more with v1.3.0.

### PR DESCRIPTION
**Pull request by ghost**
Adds v1.3.0 examples to match current functions and information. Partially a fix for #11. Still need to write in-depth docs and actual examples of usage.

### CODE REVIEW

**Comment by iamperry294** [comment]
@AmazingMech2418

File: README.md [hunk 1]
```diff
@@ -3,53 +3,77 @@
 ScratchCloud is a new, powerful, Scratch cloud data API for Node.js written by AmazingMech2418! 
 
 ## Features
-- Scratch login session handling
+- Scratch login session handling (no logouts)
```
Comment (AmazingMech2418) [inline README.md:6]: There are logouts.

File: README.md [hunk 2]
```diff
@@ -3,53 +3,77 @@
...
 - Cloud data manipulation
 - Cloud data reading
 - Async/await options (using promises)
-- Ability to wait until a target variable is a certain value (async only)
+- Ability to wait until a target variable is less than, greater than, not equal, or equal to a value
+- Onload function to wait for a cloud variable to be loaded, for stability
+- Built in functions to mimic Scratch's cloud variable functions
+- Shows cloud changes that are NOT from your node program
```
Comment (AmazingMech2418) [inline README.md]: Emphasis on "not" not needed; also, "Node.js" preferred to "node".

File: README.md [hunk 3]
```diff
@@ -3,53 +3,77 @@
...
 ## Examples
 
 ### Callbacks
 ```js
 // Require module
-const {Session, Cloud} = require('scratchcloud');
+const { Session, Cloud } = require('scratchcloud');
 
 // Create user session
+const project = 458027255 // Project ID
```
Comment (AmazingMech2418) [inline README.md]: Semicolon

File: README.md [hunk 4]
```diff
@@ -3,53 +3,77 @@
...
 const session = new Session(process.env.USERNAME, process.env.PASSWORD, function(user) {
   // Create cloud session
-  const cloud = new Cloud(user, 458027255, function(error, c) {
+  const cloud = new Cloud(user, project, function(error, cloud) {
     // On error, throw error
-    if(error)throw error;
+    if(error) throw error;
+    
+    // Wait until cloud variables have been loaded
+    cloud.onload("cloud");
```
Comment (AmazingMech2418) [inline README.md:31]: Only works for async.

File: README.md [hunk 5]
```diff
@@ -3,53 +3,77 @@
...
 
     // On change of cloud variable (not including by your Node program), display change
-    c.on('set', function(name, value) {
+    cloud.on('set', function(name, value) {
       console.log(name, value);
     });
-  })
+    
+    // Wait until a cloud variable is a specific value
+    cloud.waitUntil("cloud", val => val > 4); // Cloud variable is greater than 4
```
Comment (AmazingMech2418) [inline README.md:48]: Only works for async.

File: README.md [hunk 6]
```diff
@@ -3,53 +3,77 @@
...
 ```js
 // Require module
-const {Session, Cloud} = require('scratchcloud');
+const { Session, Cloud } = require('scratchcloud');
 
 // Async function to enable async/await
 (async () => {
   // Create user session
   const user = await Session.createAsync(process.env.USERNAME, process.env.PASSWORD);
+  const project = 458027255 // Project ID
```
Comment (AmazingMech2418) [inline README.md]: Semicolon again

File: README.md [hunk 7]
```diff
@@ -59,9 +83,15 @@ const {Session, Cloud} = require('scratchcloud');
...
-  // Wait until cloud variable is 10
-  await cloud.waitUntil('cloud', 10);
+    
+  // Wait until a cloud variable is a specific value
+  await cloud.waitUntil('cloud', 10);  // Wait until cloud variable is 10
+  await cloud.waitUntil("cloud", val => val > 4); // Cloud variable is greater than 4
+  await cloud.waitUntil("cloud", val => val !== 6); // Cloud variable isn't set to 6
+  
+  // Change cloud variable by 5
+  await cloud.changeBy("cloud", 5);
```
Comment (AmazingMech2418) [inline README.md]: No need for `await`.

File: README.md [hunk 8]
```diff
@@ -70,8 +100,7 @@ const {Session, Cloud} = require('scratchcloud');
 
 
 ## Other Information
-You can use the `cloud.get(variableName)` to get the value of a cloud variable by name.
```
Comment (AmazingMech2418) [inline README.md:73]: Not demonstrated

**Review by AmazingMech2418 — CHANGES_REQUESTED** [review]
A few changes needed...

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md:6]: I don't seem to get logged out anymore though.

File: README.md (hunk 4 above)
Comment (ghost) [reply README.md:31]: Technically, you *could* use it with callbacks.

File: README.md (hunk 1 above)
Comment (iamperry294) [reply README.md:6]: @AmazingMech2418

Comment (ghost) [reply README.md:6]: @AmazingMech2418

**Comment by ghost** [comment]
@AmazingMech2418 Is this still valid?

**Pull request closed without merging**